pkg/controllers/dashboard/systemcharts/controller_test.go

```diff
@@ -191,8 +188,14 @@ func Test_ChartInstallation(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			ctrl := gomock.NewController(t)
-			namespaceCtrl := NewMockNamespaceController(ctrl)
+			namespaceCtrl := fake.NewMockNonNamespacedControllerInterface[*v1.Namespace, *v1.NamespaceList](ctrl)
 			namespaceCtrl.EXPECT().Delete(operatorNamespace, nil).Return(nil)
+			configCache := fake.NewMockCacheInterface[*v1.ConfigMap](ctrl)
```

Comment:
Same as above, don't need a new controller for each test

Reply:
We need a new gomock.Controller per `*testing.T`